authors/settings.py

```diff
@@ -85,7 +86,7 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': os.environ.get('DB_NAME', 'ahdb'),
         'USER': os.environ.get('DB_USER', 'postgres'),
-        'PASSWORD': os.environ.get('PASSWORD', ''),
+        'PASSWORD': os.environ.get('PASSWORD', 'moschinogab19'),
```

Comment:
I think as we had agreed the password for the local database config should be kept open.

Reply:
@billkabanga ,was this resolved?

Reply:
No this was not resolved. We will have to fix it in this sprint.

Reply:
Guys let me resolve this on Monday morning, I forgot of that when we were pushing.